tests/measurements/test_counts.py

```diff
@@ -145,19 +145,29 @@ def test_counts_with_nan_samples(self):
         total_counts = sum(count for count in result.values())
         assert total_counts == 997
 
-    @pytest.mark.parametrize("n_wires", [5, 8, 10])
-    @pytest.mark.parametrize("all_outcomes", [True, False])
-    def test_counts_multi_wires_no_overflow(self, n_wires, all_outcomes):
+    @pytest.mark.parametrize("batch_size", [None, 1, 4])
+    @pytest.mark.parametrize("n_wires", [4, 10, 65])
+    @pytest.mark.parametrize("all_outcomes", [False, True])
+    def test_counts_multi_wires_no_overflow(self, n_wires, all_outcomes, batch_size):
         """Test that binary strings for wire samples are not negative due to overflow."""
+        if all_outcomes and n_wires == 65:
+            pytest.skip("Too much memory being used, skipping")
```

Comment:
We could think about adding a corresponding warning to `qml.counts`, I suppose? :sweat_smile: